components/dropdown/dropdown.js

```diff
@@ -0,0 +1,30 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { DropdownOpenerMixin } from './dropdown-opener-mixin.js';
+import { dropdownOpenerStyles } from './dropdown-opener-styles.js';
+
+class Dropdown extends DropdownOpenerMixin(LitElement) {
+
+	static get properties() {
+		return {};
+	}
+
+	static get styles() {
+		return [dropdownOpenerStyles];

```

Comment:
Nit: I don't think this needs to be an array.  I think we can just return `dropdownOpenerStyles` in this case.